legend-depot-store-notifications/src/main/java/org/finos/legend/depot/store/notifications/api/Queue.java

```diff
@@ -27,7 +27,7 @@
 
     List<MetadataNotification> pullAll();
 
-    Optional<MetadataNotification> getFirstInQueue();
+    Optional<MetadataNotification> getFirstInQueueByPriority();
```

Comment:
lets leave the name of the method as ... the implementation of what first means could change